07-practical-work-1/COURSE_MATERIAL.md

```diff
@@ -238,7 +272,8 @@ La note a été saisie dans GAPS également.
 
 Nous restons à votre disposition pour toute question.
 
-Cordialement, Ludovic Delafontaine et Hadrien Louis
+Cordialement,
+Ludovic Delafontaine et Hadrien Louis
```

Comment:
We should also add Géraud. To do this, I suggest adding a more generic placeholder: [TAs names] (TAs: teaching assistant names) for example.